### ISSUE TITLE
BURNDOWN диаграмма 

### ISSUE THREAD

**Issue by BlizD** [BlizD/Tasks#44]
BURNDOWN диаграмма 

Чтобы ее сделать необходимо:
1. Сохранять в периодическом регистре сведений статусы задач; 
1. В спринты добавить ТЧ "задачи" при этом исправить существующие методы. В этой тч необходимо указывать плановые даты. 
1. Выбрать как лучше реализовать диаграмму: средствами 1с или JavaScript диаграммы 

Если будут желающие опишу более подробно.

**Comment by JohnyDeath** [comment]
В гитлабах/зенхабах такие диаграммы строятся для Milestones (Этап, веха, рубеж, контрольная точка..) , а не для проекта в целом. И это кажется более логичным, тем более если мы позиционируемся как "гибкие разработчики".

Кратко суть:

1. Добавляем Этап, в котором указываем его срок
1. В задачах указываем Этап, в котором она должна быть выполнена
1. Для такой диаграммы важно знать только то, что задача закрыта или еще нет и к какому Этапу она принадлежит.

До кучи где-нибудь приятно смотреть на примерно такое:
![Прогресс](https://docs.gitlab.com/ce/user/project/milestones/img/progress.png)

**Comment by JohnyDeath** [comment]
И саму картинку Burndown диаграммы оставлю здесь:

![Burndown диаграмма](https://git.rootservices.net/help/user/project/milestones/img/burndown_chart.png)

**Comment by capitoshko** [comment]
Я такое точно сделаю - простенький график для скрам - команды.

**Comment by BlizD** [comment]
@capitoshko вот это было бы круто. Но сначало надо исправить хранение спринтов, как мне кажется.

**Comment by capitoshko** [comment]
Согласен. Не не принципиально. Есть время задач в спринте. Есть закрытые задачи, есть незакрытые. От этого и строить. Это если не ошибаюсь

**Comment by BlizD** [comment]
Да согласен

**Comment by capitoshko** [comment]
Для плановой оценки задачи предлагаю добавить в  справочник Задачи новый реквизит "Плановая оценка задачи". Пока заполнять вручную. Это нужно для начальной точки диаграммы сгорания спринта, чтобы получить оценку задач в спринте.

**Comment by capitoshko** [comment]
Выложил файл с конфигурацией в задаче #86

**Comment by BlizD** [comment]
Залил изменения, последний дев CF лежит в папке.
https://drive.google.com/open?id=1PMLkmLSceiseHCCqN2hv2wAB0N8p7u7m

**Comment by capitoshko** [comment]
Есть ошибки в релизе

В списке задач ошибка

Ошибка при выполнении запроса.
по причине:
{(19, 21)}: Поле не найдено "СправочникузЗадачи.Спринт"
СправочникузЗадачи.<<?>>Спринт,

**Comment by capitoshko** [comment]
Ошибку исправил

https://drive.google.com/file/d/1hW9mRp0JoqHUthjHS4b8HCVKclKcNwA0/view?usp=sharing

**Comment by capitoshko** [comment]
Буду на месте и посмотрю. Если не ошибаюсь, то нужен заполненный производственный кадендарь

**Comment by BlizD** [comment]
@capitoshko 
* Необходимо выдавать сообщение об этом, что его надо заполнить, если он не заполнен.
У себя заполнил, все равно пустое показывает.

**Comment by BlizD** [comment]
У меня была проблема из за следующего:
* не заполнена "Дата окончания" в спринте;
* не заполнены ЧасыПлан

**Comment by capitoshko** [comment]
Ок. Я попробую добавить сообщения
